### ISSUE TITLE
npm run dev error 

### ISSUE THREAD

**Issue by stevesloka** [Belar/hapi-vue#1]
Hey I'm playing around with the project. After I `npm install` and then `npm run dev` I get the following error, have you run into before?

```
ERROR in ./client/main.js
Module build failed: TypeError: fileSystem.statSync is not a function
    at module.exports (/Users/slokas/dev/vuetest/app2/jupiter/node_modules/babel-loader/lib/utils/exists.js:7:25)
    at find (/Users/slokas/dev/vuetest/app2/jupiter/node_modules/babel-loader/lib/resolve-rc.js:13:9)
    at Object.module.exports (/Users/slokas/dev/vuetest/app2/jupiter/node_modules/babel-loader/lib/index.js:113:132)
 @ multi main
```

**Comment by stevesloka** [comment]
Seems to be something with the package version of `babel-loader`. Using this version it builds: ` "babel-loader": "^6.0.0"`

**Comment by Belar** [comment]
Hi, thank you for reporting the bug.

You are right, the issue starts with `babel-loader`, exactly in `@7.1.2` and it using `statSync`, which has been introduced in `webpack@2.2.0`. Although I don't "fix" versions in the template, and general dependency goes (as of today) with `webpack@2.7.0`, there is `hapi-webpack-dev-middleware` dependency which requires fixed `webpack@2.1.0-beta25`; causing use of outdated webpack.

Current workaround is to use `babel-loader@7.1.1`.

**Comment by Belar** [comment]
Fixed in https://github.com/Belar/hapi-vue/commit/6f6d6f59fc0a0d201b23d979f7254bf9ee0c7fb7.
